tests/system/data_sources/test_bigquery.py

```diff
@@ -1147,18 +1163,28 @@ def test_bigquery_generate_table_partitions(mock_conn):
             "-pk=course_id,quarter_id,student_id",
             "-hash=*",
             "-cdir=/home/users/yaml",
-            "-pn=4",
+            "-pn=9",
+            "-ppf=5",
+            "-filters=quarter_id != 1111",
         ]
     )
     config_managers = main.build_config_managers_from_args(args, consts.ROW_VALIDATION)
     partition_builder = PartitionBuilder(config_managers, args)
```

Comment:
This test and it's assertions are repeated for all DB types. Can we move the assertions to [common_functions](https://github.com/GoogleCloudPlatform/professional-services-data-validator/blob/develop/tests/system/data_sources/common_functions.py)?

Reply:
Yes, I realize that - the only thing that is likely different is the EXPECTED_PARTITION_FILTER - which can be an argument. I was thinking this sort of cleanup is something we should do for the whole test suite.